src/others/snapshot/lv_snapshot.h

```diff
@@ -67,33 +67,33 @@ lv_result_t lv_snapshot_reshape_draw_buf(lv_obj_t * obj, lv_draw_buf_t * draw_bu
 lv_result_t lv_snapshot_take_to_draw_buf(lv_obj_t * obj, lv_color_format_t cf, lv_draw_buf_t * draw_buf);
 
 /**
- * Legacy API, use `lv_draw_buf_destroy` instead.
+ * @deprecated Use `lv_draw_buf_destroy` instead.
  *
  * Free the snapshot image returned by @ref lv_snapshot_take
  * @param dsc   the image descriptor generated by lv_snapshot_take.
  */
 static inline void lv_snapshot_free(lv_image_dsc_t * dsc)
 {
-    LV_LOG_WARN("Legacy API, use lv_draw_buf_destroy instead.");
+    LV_LOG_WARN("Deprecated API, use lv_draw_buf_destroy directly.");
     lv_draw_buf_destroy((lv_draw_buf_t *)dsc);
 }
 
 /**
- * Legacy API, use lv_snapshot_take_to_draw_buf instead.
  * Take snapshot for object with its children, save image info to provided buffer.
  * @param obj       the object to generate snapshot.
  * @param cf        color format for generated image.
  * @param dsc       image descriptor to store the image result.
  * @param buf       the buffer to store image data. It must meet align requirement.
  * @param buf_size  provided buffer size in bytes.
  * @return          LV_RESULT_OK on success, LV_RESULT_INVALID on error.
+ * @deprecated      Use lv_snapshot_take_to_draw_buf instead.
```

Comment:
In some places `@deprecated` is the first comment line, here it is simply added after `@return`

Reply:
I think generated doc will include `deprecated` note, no matter where it locates. Let's see if it's true.